README.md

```diff
@@ -6,6 +6,7 @@ Ballerina Constraint Library
   [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-constraint.svg?label=Last%20Commit)](https://github.com/ballerina-platform/module-ballerina-constraint/commits/master)
   [![GitHub issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/constraint.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fconstraint)
   [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-constraint/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-constraint) 
+  [![GraalVM Check](https://img.shields.io/github/workflow/status/ballerina-platform/module-ballerina-constraint/Build%20with%20bal%20test%20native?label=GraalVM%20Check)](https://github.com/ballerina-platform/module-ballerina-constraint/actions/workflows/build-with-bal-test-native.yml)
```

Comment:
Shall we re-order these? I prefer the following order:

1. Build
2. CodeCov
3. Trivy
4. GraalVM Check
5. Last Commit
6. GitHub Issues

Reply:
Fixed via [7204544](https://github.com/ballerina-platform/module-ballerina-constraint/pull/33/commits/7204544eb76aae55c0959a9f3820d35fa02d2746)